src/Integration/MefServices/ActiveSolutionTracker.cs

```diff
@@ -55,6 +59,10 @@ public ActiveSolutionTracker([Import(typeof(SVsServiceProvider))] IServiceProvid
         #region IVsSolutionEvents
         int IVsSolutionEvents.OnAfterOpenProject(IVsHierarchy pHierarchy, int fAdded)
         {
+            if (folderWorkspaceService.IsFolderWorkspace())
```

Comment:
Technically we know from the given pHierarchy that it's a folder, but I thought it would be more consistent to use the existing `folderWorkspaceService`.

Reply:
Probably worth adding a comment as to why we need to raise this event for folder workspaces, as it's not obvious.